packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -284,6 +284,7 @@ async function registerTestView(): Promise<vscode.Disposable> {
 }
 
 export async function deactivate() {
+  await languageClient?.stop();
```

Comment:
sweet.  We already have a unit test suite for this class.  Mind adding a unit test for the deactivate function?  I'll QE in the mean time.

Reply:
Will do.